forms-flow-web/src/components/Modeler/Editors/BpmnEditor/index.js

```diff
@@ -82,6 +98,56 @@ const BpmnEditor = forwardRef(({ bpmnXml, setLintErrors }, ref) => {
     handleImport: (bpmnXml) => handleImport(bpmnXml),
   }));
 
+  
+ 
+  const validateProcess = async (xml) => {
+    // If the BPMN Linting is active then check for linting errors, else check for Camunda API errors
+    // Check for linting errors in the modeler view
```

Comment:
don't write any function inside this component please make sure the bpmn editor is reusable component only pass the xml and error setting function, and you can pass the ref and take the xml data by calling getBpmnModler function using this ref @abilpraju-aot  so please rework this part